samples/python/whisper_speech_recognition/README.md

```diff
@@ -15,9 +15,18 @@ optimum-cli export openvino --trust-remote-code --model openai/whisper-base whis
 
 ## Prepare audio file
 
-Prepare audio file in wav format with sampling rate 16k Hz.
+You can prepare an audio file in WAV format with a sampling rate of 16k Hz using the `recorder.py` script. The script will record 5 seconds of audio from the microphone. 
```

Comment:
`recorder.py` isn't added

Reply:
I see now. The script is in https://github.com/openvinotoolkit/openvino.genai/pull/1010